docs/testplan/MACsec-test-plan.md

```diff
@@ -505,3 +505,10 @@ The switch should only react encrypted PFC frames, send encrypted PFC frames.
 #### Macsec enabled on all interfaces and the DUT is rebooted
 
 - Check the macsec docker comes up and macsec sessions are established.
+
+#### Verify mismatched primary Macsec key reboot behavior
```

Comment:
Thanks for adding this test. Here too I wanted to point out a similar test which is already present macsec testsuite **- test_mismatch_macsec_configuration** : https://github.com/sonic-net/sonic-mgmt/blob/fa5ab2566c9e30a6cbdaab87345eadd9ae63e7b2/tests/macsec/test_fault_handling.py#L95. 
We needn't really do a reboot to verify that macsec sessions are not coming up with a wrong/corrupt key.

Reply:
@Azarack , your test targets on the same scenario as the test mentioned above. I think we should close this PR to avoid duplication.

Reply:
Judy indicated that the reboot operation does not make a fundamental difference. Can you explain why the reboot is necessary here?